### ISSUE TITLE
twisted.conch.ssh.factory's docstring slightly inaccurately claims to contain OpenSSH support

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8990]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#8990|
|Type|defect|
|Created|2017-01-12 17:41:36Z|

https://github.com/twisted/twisted/blob/e116bbe34dc3a3855a674043bc5f30d7e6417c7f/src/twisted/conch/ssh/factory.py#L5-L6 seems to hint that there's an OpenSSHFactory in the module, but that actually lives at twisted.conch.openssh_compat.factory.OpenSSHFactory

<details><summary>Searchable metadata</summary>

```
trac-id__8990 8990
type__defect defect
reporter__Julian Julian
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1484242896332241 1484242896332241
changetime__1484611647686690 1484611647686690
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @rodrigc|
|-|-|

OK.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

Fixed in [3257fbb3504329f603cc9b6e17442d913170b5d1](http://twistedmatrix.com/trac/changeset/3257fbb3504329f603cc9b6e17442d913170b5d1)

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc set status to `closed`|
|-|-|
